pkg/project/registry/project/proxy/proxy.go

```diff
@@ -154,19 +158,38 @@ func (s *REST) Update(ctx kapi.Context, name string, objInfo rest.UpdatedObjectI
 		return nil, false, fmt.Errorf("not a project: %#v", obj)
 	}
 
-	s.updateStrategy.PrepareForUpdate(ctx, obj, oldObj)
-	if errs := s.updateStrategy.ValidateUpdate(ctx, obj, oldObj); len(errs) > 0 {
```

Comment:
If you get a validation error and it is in `metadata`, you can try the impersonating update on the namespace.

Otherwise, you can escalate using your privileged client.